telco-ran/configuration/kube-compare-reference/required/cluster-tuning/monitoring-configuration/ReduceMonitoringFootprint.yaml

```diff
@@ -11,5 +11,31 @@ data:
       enabled: false
     telemeterClient:
       enabled: false
+    nodeExporter:
+      collectors:
+        buddyinfo: {}
+        cpufreq: {}
+        ksmd: {}
+        mountstats: {}
+        netclass: {}
+        netdev: {}
+        processes: {}
+        systemd: {}
+        tcpstat: {}
     prometheusK8s:
-       retention: 24h
+      additionalAlertmanagerConfigs:
+      - apiVersion: v2
+        bearerToken:
+          key: token
+          name: observability-alertmanager-accessor
+        scheme: https
+        staticConfigs:
+        - (?<alertmanager_endpoint>.*)
```

Comment:
I'm not clear on what the regex gives us here?  If the goal is to allow the user to set any value, we can just template it without a regex.

Reply:
User can set any value, whether it is a valid or invalid value, thats rests on user. The only thing we have to make sure is that it is not empty.